.circleci/check-testnet-deployment.js

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env node
+
+const { waitUntilSync, getBlockHeight, getCommit } = require('@orbs-network/orbs-nebula/lib/metrics');
+const fetch = require('node-fetch');
+
+const TOPOLOGY = 'https://s3.us-west-2.amazonaws.com/boyar-testnet-bootstrap/boyar/config.json';
+const TARGET_HASH = process.argv[2];
+
+if (!TARGET_HASH) {
+    console.log('No target hash to check');
+    process.exit(1);
+}
+
+async function getTopology() {
+    const result = await fetch(TOPOLOGY);
```

Comment:
You have already downloaded it with curl, why do it here again? I would prefer modifying the local file.

Reply:
Agreed - changing it.